.github/workflows/scripts/update-eventuals-git-repository.sh

```diff
@@ -0,0 +1,77 @@
+# !/bin/bash
+
+# File passed to this script as an argument in which we gonna
+# update 'git_repository' bazel rule for eventuals 
+# (WORKSPACE.bazel usually).
+file=$1
+
+# The last commit for eventuals.
+commit=$2
+
+# The latest shallow_since for eventuals.
+shallow_since=$3
+
+if [[ ! -f "$file" ]]; then
+    echo "$file doesn't exist"
+    exit 1
+fi
+
+if [[ ${#commit} == 0 ]]; then
+    echo "Incorrect commit value."
```

Comment:
Let's be consistent no periods!
```suggestion
    echo "Missing or invalid commit value"
```